etc/build_template.config

```diff
@@ -0,0 +1,17 @@
+# Specify the default architecture used by kw
```

Comment:
I recently introduced the `kw init --template` and this commit will be affected by this feature. You will also need to update the template feature to copy the `build.config` file correctly.

1. https://kworkflow.org/man/features/init.html